### PR TITLE
add NL.svg & nl.json to langs

### PR DESCRIPTION
**Pull request by Phill030**

### CODE REVIEW

File: langs/nl.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      "HOME": "Homepage",
+      "WELCOME_TO": "Welkom op\n{0}",
+      "RECENT_SERVERS": "Laatste servers",
+      "PLAYERS_ONLINE:ONE": "{0} speler online",
+      "PLAYERS_ONLINE:OTHER": "{0} spelers online",
+      "SERVERS_ONLINE:ONE": "{0} server online",
+      "SERVERS_ONLINE:OTHER": "{0} server online",
+      "SERVERS": "Server",
+      "FAVORITES": "Favorieten",
+      "HISTORY": "Cursus",
```
Comment (C0kkie) [inline langs/nl.json:14]: Are you sure its the correct translation? Deepl and google says its wrong

File: langs/nl.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      "HIDE": "Verbergen",
+      "FULL": "Vol",
+      "EMPTY": "Leeg",
+      "LOCKED": "Wachtwoord beschermd",
+      "SERVER_NAME": "Servernaam",
+      "PLAYERS": "Spelers",
+      "GAMEMODE": "Spelwijze",
+      "PING": "Ping",
+      "WEBSITE": "Website",
+      "FAVORITE_SERVERS": "Favorieten",
```
Comment (C0kkie) [inline langs/nl.json:31]: Maybe translate it like the english one that it will be "Favorite servers"

File: langs/nl.json (hunk 1 above)
Comment (Phill030) [reply langs/nl.json:14]: Or which line are you talking about?

**Comment by C0kkie** [comment]
Thanks for you contribution :)

**Pull request merged**